### ISSUE TITLE
Memory leak

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/geometric-optics#400]
For https://github.com/phetsims/qa/issues/787.
Memory test results on Win 11 Chrome
Start: 42.7 MB
1 min: 87.0
2 min: 114
3 min: 147
4 min: 176
5 min: 200
6 min: 232
7 min: 264
8 min: console crashed

**Comment by pixelzoom** [comment]
Hmmm.... That's very surprising.  About a week ago, I did extensive memory testing for brand=phet in https://github.com/phetsims/geometric-optics/issues/372 and brand=phet-io in https://github.com/phetsims/geometric-optics/issues/373.  There is no dynamic memory allocation in this sim. Everything except the Options dialog is allocated at startup. And the Options dialog is allocated  when you first request it, then reused.

I'll try to reproduce with https://phet-dev.colorado.edu/html/geometric-optics/1.1.0-rc.1/phet/geometric-optics_all_phet.html, which I'm assuming is what you were running.

**Comment by pixelzoom** [comment]
Reproduced in 1.1.0-rc.1.  Here are the heap snapshots. Snapshot 1 is startup, after visiting all screens.  The other snapshots are at 1-minute intervals with `?fuzz`.

I'm guessing that a leak has been introduced in some very-active piece of common code.

<img width="252" alt="screenshot_1628" src="https://user-images.githubusercontent.com/3046552/159557721-cb8d1a47-121d-4e42-952a-f4007cdc1e2e.png">

**Comment by pixelzoom** [comment]
In Slack, @KatieWoe said:
> https://github.com/phetsims/qa/issues/789 also has a memory leak, though it is smaller. It was built at about the same time, so you may be able to work with CK if they are related.

**Comment by pixelzoom** [comment]
@jonathanolson identified https://github.com/phetsims/scenery/issues/1356 as a possible source of the leak.

**Comment by jessegreenberg** [comment]
Sorry about that! The leak has been fixed in https://github.com/phetsims/scenery/issues/1356#issuecomment-1080856852. Commit https://github.com/phetsims/scenery/commit/6d4ecc64393c96c4ce744a6e77d9e55c036499cd is what will need to be patched into a "geometric-optics-1.1" branch of scenery.

**Comment by pixelzoom** [comment]
1.1 branch patched and tested using the steps below.  Verified that the memory leak is fixed. Ready for testing in next 1.1 RC.

```
% cd perennial 
% grunt checkout-target --repo=geometric-optics --target=1.1
% cd scenery
% git checkout -b geometric-optics-1.1
% git cherry-pick 6d4ecc64393c96c4ce744a6e77d9e55c036499cd
% cd geometric-optics
% grunt --minify.mangle=false --brands=phet
// did a memory test, results shown below
% cd scenery
% git push origin geometric-optics-1.1
% cd geometric-optics
% cp build/phet/dependencies.json .
% git add dependencies.json
% git commit -m https://github.com/phetsims/geometric-optics/issues/400
% git push origin 1.1
```

<img width="262" alt="screenshot_1637" src="https://user-images.githubusercontent.com/3046552/160477573-6c5beb8c-b265-4fe7-ba27-646681d53bc1.png">

**Comment by pixelzoom** [comment]
To verify in 1.1.0-rc.2, do a heap comparison test. The heap snapshots should look similar in size to https://github.com/phetsims/geometric-optics/issues/400#issuecomment-1081083988.

Please close after verifying.
